catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -374,9 +374,17 @@
                             com.vividsolutions.jts.simplify,
                             com.vividsolutions.jts.util,
                             <!-- Remember, this is the same as saying, 2.0 or higher, not solely 2.0 -->
+                            ddf.action,
+                            ddf.catalog,
+                            ddf.catalog.event,
+                            ddf.catalog.pubsub,
+                            ddf.catalog.transform,
                             ddf.catalog.data;version="2.0",
```

Comment:
Either add `;version-"2.0",`  to the new ddf.* packages or remove it from all of the old ddf.* imports and remove the comment above.